sebs/faas/function.py

```
@@ -234,7 +248,9 @@ def _http_invoke(self, payload: dict, url: str, verify_ssl: bool = True) -> Exec
                 self.logging.error("Output: {}".format(data.getvalue().decode()))
             else:
                 self.logging.error("No output provided!")
-            raise RuntimeError(f"Failed invocation of function! Output: {data.getvalue().decode()}")
+            raise RuntimeError(
+                f"Failed invocation of function! Output: {data.getvalue().decode()}"
+            )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)